kubeflow/application/OWNERS

```diff
@@ -0,0 +1,5 @@
+approvers:
+  - jlewi 
+  - kkasravi
+reviewers:
+  - jlewi
```

Comment:
nit: users shouldn't appear as both approvers and reviewers; approvers are always considered reviewers.